app/response_processor.py

```diff
@@ -75,13 +76,14 @@ def process(self, msg, tx_id=None, decrypt=True):
             self.logger.info("Invalid survey data, skipping receipting and downstream processing")
             decrypted_json['invalid'] = True
 
-        self.store_survey(decrypted_json)
+        id_tag = self.store_survey(decrypted_json)
+        self.logger.info("id_tag: {}".format(id_tag))
```

Comment:
Few things here:
- f-strings >>>> `.format` (self.logger.info("id_tag: {}".format(id_tag)) would become self.logger.info(f"id_tag: {id_tag}")
- Use structured logging, makes it easier to search on field. (self.logger.info("id_tag: {}".format(id_tag)) would become something like self.logger.info("Saved data to the database".format(id_tag), id=id_tag)
- The string in a log line should be informative.  What you have here isn't informative.  The example I put previously with something like 'Saved data to the database' actually says what happened with some context, so it can be useful later.

Putting that all together, this log line should be something like:
```python
self.logger.info("Saved data to the database", id=id_tag)
```